app/extensions/brave/content/scripts/pageInformation.js

```diff
@@ -59,6 +59,14 @@
           expr['arguments'].forEach((argument) => { args.push(traverse(argument)) })
           return value[traverse(op1.property, true)].apply(value, args)
 
+        case 'NewExpression':
+          op1 = expr.callee
+          if (op1.type !== 'Identifier') throw new Error('unexpected callee: ' + op1.type)
+          if (op1.name !== 'Set') throw new Error('only new Set(...) is allowed, not new ' + op1.name)
+          args = []
+          expr['arguments'].forEach((argument) => { args.push(traverse(argument)) })
+          return new Set(args[0])
```

Comment:
`Set` takes at most one argument, which is an array. if i were to pass `args` instead of `args[0]`, then i would need to do use `.apply()` as well. i will make it clearer.